ui/page/staking/overview.go

```diff
@@ -593,14 +602,7 @@ func (pg *Page) startTicketBuyerPasswordModal() {
 			}
 
 			go func() {
-				vsp, err := pg.WL.MultiWallet.NewVSPClient(tbConfig.VspHost, pg.ticketBuyerWallet.ID, uint32(tbConfig.PurchaseAccount))
-				if err != nil {
-					pg.Toast.NotifyError(err.Error())
-					pm.SetLoading(false)
-					return
-				}
-
-				err = vsp.StartTicketBuyer(tbConfig.BalanceToMaintain, []byte(password))
+				err := pg.ticketBuyerWallet.StartTicketBuyer(tbConfig.VspHost, nil, tbConfig.PurchaseAccount, tbConfig.BalanceToMaintain, []byte(password))
```

Comment:
I'm going to modify this to only take the password. The other values can be gotten from wallet config.